settings/home_unix_test.go

```diff
@@ -0,0 +1,36 @@
+// +build linux darwin
+
+package settings
+
+import (
+	"os"
+	"testing"
+
+	"github.com/arschles/assert"
+)
+
+// TestFindHome ensures the correct home directory is returned by FindHome().
+func TestFindHome(t *testing.T) {
+	previousHome := os.Getenv("HOME")
+
+	expectedHomeDir := "/d/e/f"
+	os.Setenv("HOME", expectedHomeDir)
+
+	assert.Equal(t, FindHome(), expectedHomeDir, "output")
+
+	// Reset HOME.
+	os.Setenv("HOME", previousHome)
```

Comment:
Is there any reason to reset at the end? ENV variables set in a program cannot effect your environment.